### PR TITLE
Autojoin battles in event of disconnection

### PR DESCRIPTION
**Pull request by xfix**

### CODE REVIEW

**Comment by xfix** [comment]
Hm, I just realized is not going to work well with server restarts. Might as well wait until I implement persistent battles.

File: js/client.js [hunk 1]
```diff
@@ -368,6 +368,18 @@
 						}
 					}
 					this.send('/autojoin ' + autojoinIds.join(','));
+
+					var battleAutojoin = (Tools.prefs('battleautojoin') || '').split(',');
```
Comment (Zarel) [inline js/client.js]: We don't need a separate `battleAutojoin`; can't we just add it to the normal autojoin list?

Comment (Zarel) [inline js/client.js]: This also might interact weirdly with opening multiple tabs; you probably don't want battles to get focused instantly.

Comment (Zarel) [inline js/client.js]: Might be more straightforward for the server to send you a list of battles you're currently in, and just make the "rejoin battle" button very prominent.

**Pull request closed without merging**